backend/server/src/db/migrations/4-series.sql

```diff
@@ -0,0 +1,20 @@
+call prepare_randomized_ids('series');
+
+create table series (
+    id bigint primary key default randomized_id('series'),
+
+    -- Opencast internal data
+    opencast_id text not null,
+    created_by text not null,
```

Comment:
What about this btw, @LukasKalbertodt, @lkiesow?

Will we ever need that? And what would we even put here, the username? What if that changes on the Opencast side? I mean sure, we will have some kind of sync anyway. :thinking: Will we be syncing users as well? And should this then be a foreign key, rather? I guess that's a question for later.

Maybe we could leave this out for now, as well? :thinking:

Reply:
I can't really say anything about that, but leaving it out seems fine for now.